src/Symfony/Component/DependencyInjection/Loader/YamlFileLoader.php

```diff
@@ -346,8 +346,13 @@ private function parseDefinition($id, $service, $file, array $defaults)
             }
 
             foreach ($service as $key => $value) {
-                if (!in_array($key, array('alias', 'public'))) {
+                if (!in_array($key, array('alias', 'public', 'deprecated'))) {
                     throw new InvalidArgumentException(sprintf('The configuration key "%s" is unsupported for the service "%s" which is defined as an alias in "%s". Allowed configuration keys for service aliases are "alias" and "public".', $key, $id, $file));
```

Comment:
Should the message be updated, too? Now it includes hardcoded `keys for service aliases are "alias" and "public"`.